INFO  | 04/15/2020 10:46:40 AM | File run_exp.py           | Line 26    | Writing log file to exp/GRAN\GRANMixtureBernoulli_transport_2020-Apr-15-10-46-39_2456\log_exp_2456.txt
INFO  | 04/15/2020 10:46:40 AM | File run_exp.py           | Line 27    | Exp instance id = 2456
INFO  | 04/15/2020 10:46:40 AM | File run_exp.py           | Line 28    | Exp comment = None
INFO  | 04/15/2020 10:46:40 AM | File run_exp.py           | Line 29    | Config =
INFO  | 04/15/2020 10:46:40 AM | File gran_runner.py       | Line 139   | Train/val/test = 8/2/2
INFO  | 04/15/2020 10:46:40 AM | File gran_runner.py       | Line 152   | No Edges vs. Edges in training set = 1.105263157894737
ERROR | 04/15/2020 10:46:53 AM | File run_exp.py           | Line 42    | Traceback (most recent call last):
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\run_exp.py", line 38, in main
    runner.train()
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\runner\gran_runner.py", line 240, in train
    data = train_iterator.next()
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\utils\data\dataloader.py", line 346, in __next__
    data = self.dataset_fetcher.fetch(index)  # may raise StopIteration
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\utils\data\_utils\fetch.py", line 47, in fetch
    return self.collate_fn(data)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\dataset\gran_data.py", line 367, in collate_fn
    constant_values=0.0) for ii, bb in enumerate(batch_pass)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\dataset\gran_data.py", line 367, in <listcomp>
    constant_values=0.0) for ii, bb in enumerate(batch_pass)
  File "<__array_function__ internals>", line 6, in pad
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\numpy\lib\arraypad.py", line 748, in pad
    pad_width = _as_pairs(pad_width, array.ndim, as_index=True)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\numpy\lib\arraypad.py", line 519, in _as_pairs
    raise ValueError("index can't contain negative values")
ValueError: index can't contain negative values

